benchmark/submit.py

```diff
@@ -0,0 +1,129 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+"""
+Submit benchmark results to a server.
```

Comment:
I kept this doc-string short, but I've now added `longdesc` field to the `Options` class.